_rules/first-focusable-element-is-link-to-main-8a213c.md

```diff
@@ -1,150 +1,255 @@
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 2
+
+The link to skip repeated content can be toggled to visible through keyboard focus.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<style>
+			#skipNav a {
+				height: 100px;
+				position: absolute;
+				top: -100px;
+			}
+			#skipNav a:focus {
+				top: 0px;
+			}
+		</style>
+		<nav id="skipNav">
+			<a href="#main-content">Skip to main content</a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Passed Example 3
+
+The link to skip repeated content has an `aria-label` to provide an accessible name.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<nav>
+			<a href="#main-content" aria-label="Skip to main content"></a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+There is no link to skip repeated content.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 2
+
+The link to skip repeated content is not the first focusable element within the `body`.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<a href="/anotherpage">A link to another page</a>
+		<nav>
+			<a href="#main-content">Skip to main content</a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 3
+
+The link to skip repeated content does not have an accessible name.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<nav>
+			<a href="#main-content"></a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 4
+
+The link to skip repeated content does not reference a valid `id` and will not move focus to the main content when activated.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<nav>
+			<a href="#invalidId">Skip to main content</a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 5
+
+The link to skip repeated content is not visible, and is not included in the accessibility tree.
+
+```html
+<!DOCTYPE html>
+<html lang="eng">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<nav>
+			<a href="#main-content" style="display:none;">Skip to main content</a>
+		</nav>
+		<!-- Repeated content -->
+		<main id="main-content">Main Content</main>
+	</body>
+</html>
+```
+
+#### Failed Example 6
+
+The link to skip repeated content does not have a semantic role of link.
```

Comment:
"The **link** to skip" is not really correct since it is, well, not a link…

And I think it would be a better example if it was actually skipping to main content (_e.g._ a button or something with a correct `onClick` property). So that we show that the problem is "not being a link" rather than "not moving focus to the correct place".

Reply:
Good catch. Wasn't sure how to describe. I will take another shot at this one and add the click event functionality